config.yaml

```diff
@@ -154,7 +153,7 @@ params:
     # Developer relevant links. These will show up on right side of footer and in the community page if you have one.
     developer:
       - name: Notary Project on GitHub
-        url: https://github.com/notaryproject/notaryproject
+        url: https://github.com/notaryproject/.github
```

Comment:
I think the original one is correct based on the name "Notary Project on GitHub"